lib/portage/dbapi/vartree.py

```diff
@@ -5563,6 +5563,16 @@ def mergeme(
                 myabsto = myabsto.lstrip(sep)
                 if self.settings and self.settings["D"]:
                     if myto.startswith(self.settings["D"]):
+                        self._eqawarn(
+                            "preinst",
+                            [
+                                _(
+                                    "QA Notice: Symlink %s points to %s inside the image directory.\n"
```

Comment:
Could we maybe clarify this a little bit? Somehow most (all?) symlinks point to somewhere inside the image directory. The problem are symlinks using parts of the image directory's path. So maybe

```suggestion
                                    "QA Notice: Symlink %s uses path of image directory to point to %s.\n"
```

Reply:
I've replaced "symlink" by "absolute symlink" which I think should clarify it.